crates/rpc/eth/logs.rs

```diff
@@ -0,0 +1,181 @@
+use crate::{
+    types::{block_identifier::BlockIdentifier, receipt::RpcLog},
+    RpcErr, RpcHandler,
+};
+use ethereum_rust_core::{H160, H256};
+use ethereum_rust_storage::Store;
+use serde::Deserialize;
+use serde_json::Value;
+use std::collections::BTreeSet;
+#[derive(Deserialize, Debug)]
+#[serde(untagged)]
+pub enum AddressFilter {
+    Single(H160),
+    Many(Vec<H160>),
+}
+
+#[derive(Deserialize, Debug)]
+#[serde(untagged)]
+pub enum TopicFilter {
+    Topic(Option<H256>),
+    Topics(Vec<Option<H256>>),
+}
+
+// TODO: This struct should be using serde,
```

Comment:
I would remove this comment. This is a more general concern, we could create a ticket

Reply:
Removed, and I've opened the [issue](https://github.com/lambdaclass/ethereum_rust/issues/449)